main.nf

```diff
@@ -228,7 +228,7 @@ workflow NFCORE_SAREK {
     versions = versions.mix(PREPARE_GENOME.out.versions)
     versions = versions.mix(PREPARE_INTERVALS.out.versions)
 
-    vep_fasta = (params.vep_include_fasta) ? fasta.map{ fasta -> [ [ id:fasta.baseName ], fasta ] } : [[id: 'null'], []]
+    vep_fasta = (params.vep_include_fasta) ? fasta.map{ fasta_ -> [ [ id:fasta_.baseName ], fasta_ ] } : [[id: 'null'], []]
```

Comment:
why is the language server adding an underscore?

Reply:
because fasta is an already defined item, so we use `_` after if the item is reused within the same scope or before is the item is dropped

Reply:
it hurts my eyes, but not going to argue with the language server